Casks/nexus-iq-cli.rb

```diff
@@ -5,14 +5,14 @@
 #
 
 cask "nexus-iq-cli" do
-  version "1.111.0-01"
-  sha256 "816e8404b9d50b277fedea9ce84e7ca9df6c54ca618889faebca098f4d6a4ad2"
-  url "https://download.sonatype.com/clm/scanner/nexus-iq-cli-1.111.0-01+183-mac.pkg"
+  version "1.113.0-02"
+  sha256 "ddcbe5f18b107cb5a544496103ac7f69e7e8d7a05ebe6b47e42b02f750d199b4"
+  url "https://download.sonatype.com/clm/scanner/nexus-iq-cli-1.113.0-02+194-mac.pkg"
   name "Nexus Native IQ CLI"
   desc "Command line utility for application scanning with Nexus IQ"
   homepage "https://help.sonatype.com/integrations/nexus-iq-cli"
   depends_on macos: ">= :mojave"
-  pkg "nexus-iq-cli-1.111.0-01 183-mac.pkg"
+  pkg "nexus-iq-cli-1.113.0-02+194-mac.pkg"
```

Comment:
the `+` symbol here needs to be removed as it's stripped from the name when downloading (url encoding dontchaknow)

Reply:
P.S. for local testing you can make these changes directly in the local Homebrew git checkout of the package (and push those changes back after testing if you like)

Reply:
interesting - cfd99a9

Reply:
> P.S. for local testing you can make these changes directly in the local Homebrew git checkout of the package (and push those changes back after testing if you like)

@kellyrob99  I assume that only applies to Mac users? In particular because of the pkg which a Linux based homebrew system wouldn't be able to handle?

Reply:
you are correct sir